### PR TITLE
Update layer 2 rollups developer doc 

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

This PR is part of a series to [audit the developer docs](https://github.com/ethereum/ethereum-org-website/issues/3908) in preparation for upgrades to our translation program.

## Notes from the changes I've suggested:

- adding more resources
- added videos explaining ZK-rollups and optimistic rollups
- improvements to readability
- updated out of date information

## Link
Deploy preview - https://deploy-preview-3996--ethereumorg.netlify.app/en/developers/docs/scaling/layer-2-rollups/

### CODE REVIEW

**Comment by minimalsm** [comment]
Open questions:
- should we be clear in the scaling section/on this page that rollups are the current winner?
- should we distinguish live on Mainnet vs not?
- should we remove projects that have lost traction?

File: src/content/developers/docs/scaling/layer-2-rollups/index.md [hunk 1]
```diff
@@ -7,21 +7,21 @@ incomplete: true
...
-You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested, and continues to be researched and developed.
+You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as rollups is an advanced topic as the technology is less battle-tested and continues to be researched and developed.
 
 ## Why is layer 2 needed? {#why-is-layer-2-needed}
 
-- Some use-cases, like blockchain games, make no sense with current transaction times
-- It can be unnecessarily expensive to use blockchain applications
+- Some use-cases, like blockchain games, make no sense with current transaction times.
+- It can be unnecessarily expensive to use blockchain applications.
 - Any updates to scalability should not be at the expense of decentralization or security – layer 2 builds on top of Ethereum.
```
Comment (wackerow) [inline src/content/developers/docs/scaling/layer-2-rollups/index.md:20]: Curious, weren't we trending towards dropping the "hard stops" at the end of bullet points?

**Review by wackerow — APPROVED** [review]
@minimalsm Made one comment, but nothing critical. Looking good!

File: src/content/developers/docs/scaling/layer-2-rollups/index.md (hunk 1 above)
Comment (minimalsm) [reply src/content/developers/docs/scaling/layer-2-rollups/index.md:20]: @wackerow The rule I've been trying to follow (which is at least normal in British English) is full stops if each list item is a complete sentence and no full stops for fragments (and don't mix full sentences with fragments in the same list).

Does that seem okay to you? I'm not wildly in favour of one way or another, just consistency.

Comment (wackerow) [reply src/content/developers/docs/scaling/layer-2-rollups/index.md:20]: Gotcha, yeah this works then.

**Review by wackerow — APPROVED** [review]

**Pull request merged**